.github/workflows/docker-pull-request.yaml

```diff
@@ -1,7 +1,10 @@
-name: PR
+name: Docker push PR
 
 on:
-  pull_request:
+  workflow_run:
+    workflows: [Python Client, App Engine, CBIR, Core, IMS, SAM, Web UI]
+    types:
+      - completed
```

Comment:
The flow you want to have is when all of the worklows have completed, then start this workflow.
However, this is not the case here, when one of them completed, this workflow will be triggered even if some of them aren't finished or haven't started yet.

From Github documentation:

_If you specify multiple `workflows` for the `workflow_run` event, only one of the workflows needs to run. For example, a workflow with the following trigger will run whenever the "Staging" workflow or the "Lab" workflow completes._

```yaml
on:
  workflow_run:
    workflows: [Staging, Lab]
    types:
      - completed
```

https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#workflow_run